src/components/UserInfo.js

```diff
@@ -2,18 +2,48 @@ export default class UserInfo {
   constructor(profileSelector) {
     this._profileElement = document.querySelector(profileSelector);
     this._nameElement = this._profileElement.querySelector(".profile__name");
-    this._jobElement = this._profileElement.querySelector(".profile__job");
+    this._aboutElement = this._profileElement.querySelector(".profile__about");
+    this._avatarElement =
+      this._profileElement.querySelector(".profile__avatar");
```

Comment:
According to the task you need to pass all 3 selectors into `UserInfo ` to search the elements


```js

class UserInfo {

  constructor({ nameSelector, jobSelector, avatarSelector }) {
    this._nameElement = document.querySelector(nameSelector);
    this.jobElement = document.querySelector(jobSelector);
    this._avatarElement = document.querySelector(avatarSelector);
  }

```